src/htmx.js

```diff
@@ -402,6 +403,17 @@ return (function () {
             }
         }
 
+        function canAccessLocalStorage() {
+            var test = 'test';
```

Comment:
Maybe something a little more htmx-specific?  `htmx:localStorageTest` or something like that?  I'd hate to accidentally overwrite something someone really wanted.

Reply:
Ah you're right :+1: ! Let me fix that right away.

Reply:
There you go @1cg , I just added a fixup commit. Thanks for your review :slightly_smiling_face: